packages/transaction_signing_gateway/lib/model/transaction_log.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:equatable/equatable.dart';
+
+class TransactionLog extends Equatable {
+  const TransactionLog({
+    required this.msgIndex,
+    required this.log,
+    this.events,
+  });
+
+  final int msgIndex;
+  final String? log;
+  final List<Object?>? events;
```

Comment:
lets prefer non-nullable types in our own models, so:
```suggestion
  final String log;
  final List<Object?> events;
```